event-stream-processing/src/shared/regex.service.ts

```diff
@@ -38,11 +43,18 @@ export class RegexService {
             // dash is usually a special value that indicates empty/missing
             continue;
           }
-          const fieldName = gropName.replace(underscoreReplaceRegex, '.');
-          lodash.set(document.data, fieldName, value);
+          if (gropName === 'extract_timestamp') {
```

Comment:
@mbystedt Should we change this to groupName?

Reply:
Yes